libctru/source/services/cfgu.c

```diff
@@ -315,3 +329,32 @@ Result CFGI_VerifySigSecureInfo(void)
 
 	return (Result)cmdbuf[1];
 }
+
+Result CFGI_SecureInfoGetSerialNumber(u8 *serial)
+{
+	Result ret = 0;
+	u32 *cmdbuf = getThreadCommandBuffer();
+
+	cmdbuf[0] = IPC_MakeHeader(0x408,1,2); // 0x4080042
+	cmdbuf[1] = 0xF;
+	cmdbuf[2] = 12 | (0xF << 4);
```

Comment:
Use [`IPC_Desc_Buffer()`](https://github.com/smealum/ctrulib/blob/master/libctru/include/3ds/ipc.h#L110) here for increased clarity.